.github/workflows/publish.yml

```diff
@@ -6,11 +6,6 @@ on:
     types:
       - created
   workflow_dispatch:
-    inputs:
-      increment:
-        description: "Use Dry Run"
-        required: true
-        default: "false"
```

Comment:
Do we need to keep the section now that it is empty?

Reply:
yes, it will allow us to trigger the job manually